docs/api.rst

```diff
@@ -65,67 +80,65 @@ API
 .. _calibration_ref:
 
 
-:mod:`tedana.combine`: Combine time series
---------------------------------------------------
+:mod:`tedana.model`: Computing TE-dependence metrics
+----------------------------------------------------
 
-.. automodule:: tedana.combine
+.. automodule:: tedana.model
    :no-members:
    :no-inherited-members:
 
-.. autosummary:: tedana.combine
+.. autosummary:: tedana.model
    :toctree: generated/
    :template: function.rst
 
-   tedana.combine.make_optcom
+   tedana.model.dependence_metrics
+   tedana.model.kundu_metrics
 
    :template: module.rst
 
-   tedana.combine
+   tedana.model.fit
```

Comment:
I'm not sure that we want to call this a `.fit`, since it could be easily confused with the sklearn convention.

Reply:
I've never liked `fit` either. We can change `tedana.model` to `tedana.metrics` (as you suggested above), but we'll still need a file below that in order to keep model/metrics as a subfolder-organized submodule. I fully expect us to expand our sets of metrics (e.g., with AROMA-like motion- and artifact-based ones), which is why I don't think we should just put it in one top-level file (i.e., `tedana/metrics.py`), but that still leaves us with the issue of what to name the file.

Reply:
Since we're not quite there yet (e.g., adding motion based metrics), though, I think we should just keep it in a top-level file for now -- we can expand when we add more. WDYT ?

Reply:
Also, #273 moves the two remaining functions (`get_coeffs` and `computefeats2`) in `model.fit` into a new stats module, so the only things in `model.fit` will be the metrics (`dependence_metrics` and `kundu_metrics`).

Reply:
I can move it there then. So, `tedana/model/fit.py` will become `tedana/model.py` and the `tedana/model/` folder will be removed. Does that sound good?